### ISSUE TITLE
Any Testing done?

### ISSUE THREAD

**Issue by Shackmeister** [iamprithvishetty/ViGimbalController#1]
Hi project looks really cool!
Have you done any general testing on it?
How well is it performing?

Last is there any way to reach you? Email Perhaps ? :)

Kindest regards
Frank Severinsen

**Comment by iamprithvishetty** [comment]
Yes I’ve done testings I’ve started it few months back and have good results, will put it up on youtube. Email me at prithvishetty08@gmail.com for further details, I would really get some help from other contributors
